modules/test/playwright/tests/calendar-web/calendarEvent.spec.ts

```diff
@@ -72,10 +73,69 @@ test('can create all-day calendar event with different time zone', async ({
 	await expect(endTime).toEqual(startTime);
 });
 
+test('can update an event with recurrence', async ({calendarWidgetPage}) => {
+	const fiveDaysFromNow = 5;
```

Comment:
Here it makes more sense to name it fiveDaysFromNow, but still, we are not even reusing this variable. I would just pass it. Or if you want to make it clearer, make `fillEventWithRecurrenceUntilDate` receive named arguments, like this:

```
fillEventWithRecurrenceUntilDate({daysFromNow: 5})
```

Reply:
For this, `fillEventWithRecurrenceUntilDate` would be defined like:

	async fillEventWithRecurrenceUntilDate({daysFromNow}: {daysFromNow: number}) {
		await this.addEventButton.click();

		await this.repeatCheckbox.setChecked(true);

		await this.modalRecurrencePage.addRecurrenceUntilDate(daysFromNow);
	}